.github/workflows/container.yml

```diff
@@ -0,0 +1,48 @@
+name: Container
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - .github/workflows/container.yml
+      - Dockerfile
+      - httpd.conf
+      - requirements
+  pull_request:
+    branches:
+      - main
+    paths:
+      - .github/workflows/container.yml
+      - Dockerfile
+
+env:
+  REGISTRY: ghcr.io
+  # This has to be lowercase
+  IMAGE_NAME: socallinuxexpo/scale-reg
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - name: Log into the container registry
+        uses: docker/login-action@v3
+        if: github.event_name != 'pull_request'
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
we might need to change something in the repo settings for this to work but we'll find out after merge